nextcord/exceptions.py

```diff
@@ -25,3 +25,7 @@ def __init__(self) -> None:
             "You have been banned by Cloudflare. "
             "See https://discord.dev/topics/rate-limits#invalid-request-limit-aka-cloudflare-bans"
         )
+
+
+class InvalidArgument(NextcordException):
```

Comment:
Why should this be used over TypeError/ValueError (also inconsistent with the rest of the errors?)

Reply:
I guess Type/ValueError could be used here yeah

Reply:
Actually this custom exception should be used, as this will be raised when 2 arguments as passed when you should only pass one of them (ie `embed` and `embeds` in  `.send()`.
Let me know or resolve this

Reply:
This should probably be better named then like ArgumentConflict or similar